CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(SOURCES
 add_library(${PROJECT_NAME} MODULE ${SOURCES})
 
 find_package(SYSREPO REQUIRED)
+find_package(LIBUCI2 REQUIRED)
 find_package(UCI REQUIRED)
```

Comment:
Can remove dependency on `uci` here, given that we have replaced UCI functionality with `uci2`?

Reply:
Yes, sorry, that was left over when I was still compiling with not all functions implemented so that other functions would work.

Reply:
Great, let's remove it! Thanks.